
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.4.0, mkdocs-material-8.5.6">
    
    
      
        <title>BGCs - Biosb Computational Metagenomics</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.20d9efc8.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  
  
    
  
  
  <meta property="og:type" content="website" />
  <meta property="og:title" content="Biosb Computational Metagenomics - BGCs" />
  <meta property="og:description" content="None" />
  <meta property="og:url" content="None" />
  <meta property="og:image" content="Noneassets/images/banner.png" />
  <meta property="og:image:type" content="image/png" />
  <meta property="og:image:width" content="1200" />
  <meta property="og:image:height" content="630" />

  <link rel="alternate" type="application/rss+xml" title="RSS feed of created content" href="Nonefeed_rss_created.xml">
  <link rel="alternate" type="application/rss+xml" title="RSS feed of updated content" href="Nonefeed_rss_updated.xml">

  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#functional-annotation-biosynthetic-gene-clusters" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Biosb Computational Metagenomics" class="md-header__button md-logo" aria-label="Biosb Computational Metagenomics" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Biosb Computational Metagenomics
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              BGCs
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/mdehollander/biosb-metagenomics" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    biosb-metagenomics
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Biosb Computational Metagenomics" class="md-nav__button md-logo" aria-label="Biosb Computational Metagenomics" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    Biosb Computational Metagenomics
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/mdehollander/biosb-metagenomics" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    biosb-metagenomics
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Day 1: Data processing
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Day 1: Data processing" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Day 1: Data processing
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/intro/" class="md-nav__link">
        Introduction
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/quality-control/" class="md-nav__link">
        Quality Control
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/assembly/" class="md-nav__link">
        Assembly
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/binning/" class="md-nav__link">
        Binning
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
    
  
  
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3" type="checkbox" id="__nav_3" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_3">
          Day 3: Functional annotation
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Day 3: Functional annotation" data-md-level="1">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Day 3: Functional annotation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../viromes/" class="md-nav__link">
        Viromics
      </a>
    </li>
  

            
          
            
              
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          BGCs
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        BGCs
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#step-1-general-overview-biosynthetic-gene-cluster-identification" class="md-nav__link">
    Step 1 -General overview: biosynthetic gene cluster identification
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-2-disease-suppressive-metabolites" class="md-nav__link">
    Step 2 - Disease-suppressive metabolites
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-3-peptidic-fragments" class="md-nav__link">
    Step 3 - Peptidic fragments
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#step-1-general-overview-biosynthetic-gene-cluster-identification" class="md-nav__link">
    Step 1 -General overview: biosynthetic gene cluster identification
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-2-disease-suppressive-metabolites" class="md-nav__link">
    Step 2 - Disease-suppressive metabolites
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-3-peptidic-fragments" class="md-nav__link">
    Step 3 - Peptidic fragments
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/mdehollander/biosb-metagenomics/edit/master/docs/functional-annotation/bgcs.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="functional-annotation-biosynthetic-gene-clusters">Functional annotation - Biosynthetic gene clusters<a class="headerlink" href="#functional-annotation-biosynthetic-gene-clusters" title="Permanent link">&para;</a></h1>
<h2 id="step-1-general-overview-biosynthetic-gene-cluster-identification">Step 1 -General overview: biosynthetic gene cluster identification<a class="headerlink" href="#step-1-general-overview-biosynthetic-gene-cluster-identification" title="Permanent link">&para;</a></h2>
<p><a href="http://bioinformatics.nl/~medem005/antismash_relaxed/index.html">Here</a> you can find antiSMASH results for the (simulated) metagenome that you analysed on Monday afternoon.
Clicking ‘Compact view’ helps to get a better overview for a metagenome output like this one.</p>
<details class="done">
<summary>1.    How many biosynthetic gene clusters (BGCs) did antiSMASH identify? Based on the results, which known compounds do you estimate this microbial community to be able to produce? Hint: take a look at the detailed knownclusterblast results for each cluster that has at least &gt; 50% similarity on the gene level to a known cluster to assess this. Do (almost) all core enzymes encoded in the gene cluster show similarity to those in the reference gene cluster?</summary>
<p>The bacillibactin, bacilysin, lichenysin and plantazolicin gene clusters look real. Others perhaps.</p>
</details>
<details class="done">
<summary>2.    One gene cluster looks like it is fragmented into two pieces. Which one? What could you do to try to assemble it into one piece?</summary>
<p>The lichenysin gene cluster. It might be possible to assemble it by studying the assembly graph or using specialized tools such as biosyntheticSPAdes.</p>
</details>
<p>Using the ‘loose’ mode (see https://antismash.secondarymetabolites.org/), multiple putative BGCs can be identified by antiSMASH, which need to be analyzed manually to assess their value. <a href="http://bioinformatics.nl/~medem005/antismash_loose/index.html">Here</a> you can find the antiSMASH results for the same genome, but now using the loose mode to predict more (putative) clusters.</p>
<details class="done">
<summary>3.    Take a look at some of the ‘newly added’ BGCs, and specifically look at the smCOG annotations and the knownclusterblast results. Can you identify some clusters that are very probable to encode the biosynthesis of an actual secondary metabolite? And can you find some clusters for which this is very unlikely? Note that some BGCs may be fragmented (visible by a note ‘Region on contig edge’) and may only show partial similarity to reference BGCs.</summary>
<p>Region 413.1 might encode a fragment of a real BGC, based on the fact that it lies on a contig edge, and the Knownclusterblast tab shows a match with part of a known BGC. Others are less likely real. Region 1068.1, for example, looks very doubtful, because it comprises only three identified enzyme-coding genes, separated by a large distance. </p>
</details>
<h2 id="step-2-disease-suppressive-metabolites">Step 2 - Disease-suppressive metabolites<a class="headerlink" href="#step-2-disease-suppressive-metabolites" title="Permanent link">&para;</a></h2>
<p>Now go back to the original results.
Beneficial plant microbiota are sometimes able to protect plants against pathogens, a microbiome-associated phenotype known as disease suppression. One of the BGCs in this metagenome that might play a role for this is found in region 550.1; this BGC shows similarity to the bacilysin BGC from Bacillus velezensis FZB42 (previously known as Bacillus amyloliquefaciens FZB42).
Look up the cluster and check out the Knownclusterblast and MIBiG comparison tabs. </p>
<details class="done">
<summary>4.    Do you think that region 550.1 is likely to encode the production of bacilysin? Why?</summary>
<p>Only the transporter-encoding gene is missing, and all enzyme-coding genes seem to be similar. So yes, it is likely.</p>
</details>
<details class="done">
<summary>5.    In the knownclusterblast result, click on the MIBiG accession number of the first hit. This will take you to the reference entry of the known bacilysin BGC in the MIBiG repository. Check out the literature references. What do you learn about the biological activity of bacilysin? How might this molecule play a role in pathogen suppression?</summary>
<p>Bacilysin is known to have antibacterial activity against bacterial phytopathogens such as Xanthomonas.</p>
</details>
<details class="done">
<summary>6.    Just downstream of the putative bacilysin gene cluster, you will find another large operon with many enzyme-coding genes. What do you think this operon might encode? Can you hypothesise a functional reason why these two operons are co-localized in the genome? Check out this paper for some hints (check section 3.4).</summary>
<p>The paper suggests that bacilysin is produced by biofilm-producing bacilli. They might use it as protective metabolites, and the bacilli might provide double protection to plant roots by forming a protective coat while also producing antimicrobials.</p>
</details>
<h2 id="step-3-peptidic-fragments">Step 3 - Peptidic fragments<a class="headerlink" href="#step-3-peptidic-fragments" title="Permanent link">&para;</a></h2>
<p>Based on a mass spectrometry experiment, an apparently new natural product is identified from the strain. Based on fragmentation analysis, it appears to be a lipopeptide. A two-amino acid-long fragment is retrieved that is reconstructed based on mass shifts from tandem mass spectra, consisting of a valine and a leucine.</p>
<p>Peptide natural products can either be produced by multimodular enzymes called nonribosomal peptide synthetases (identified as ‘NRPS’ by antiSMASH) or through ribosomal synthesis and subsequent posttranslational modification (e.g., lanthipeptides, lasso peptides, thiopeptides, etc.). For both, chemical structure predictions may be provided (see the ‘NRPS/PKS modules’ or ‘Lanthipeptide/Lassopeptide’ tabs).</p>
<details class="done">
<summary>7.    Based on the antiSMASH results, which gene cluster do you think is most likely responsible for the biosynthesis of the peptide? What strategy did you use to find this out?</summary>
<p>Region 1023.1. Two subsequent predicted amino acid substrates visible in the 'NRPS/PKS modules' tab match the monomers observed from the MS data.</p>
</details>

  <hr>
<div class="md-source-file">
  <small>
    
      Last update:
      <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">October 12, 2022</span>
      
    
  </small>
</div>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
        
        <a href="../viromes/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Viromics" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              Viromics
            </div>
          </div>
        </a>
      
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.expand"], "search": "../../assets/javascripts/workers/search.5bf1dace.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
  
      <script src="../../assets/javascripts/bundle.078830c0.min.js"></script>
      
    

  </body>
</html>