
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.4.0, mkdocs-material-8.5.6">
    
    
      
        <title>Viromics - Biosb Computational Metagenomics</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.20d9efc8.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  
  
    
  
  
  <meta property="og:type" content="website" />
  <meta property="og:title" content="Biosb Computational Metagenomics - Viromics" />
  <meta property="og:description" content="None" />
  <meta property="og:url" content="None" />
  <meta property="og:image" content="Noneassets/images/banner.png" />
  <meta property="og:image:type" content="image/png" />
  <meta property="og:image:width" content="1200" />
  <meta property="og:image:height" content="630" />

  <link rel="alternate" type="application/rss+xml" title="RSS feed of created content" href="Nonefeed_rss_created.xml">
  <link rel="alternate" type="application/rss+xml" title="RSS feed of updated content" href="Nonefeed_rss_updated.xml">

  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#functional-annotation-viromics" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Biosb Computational Metagenomics" class="md-header__button md-logo" aria-label="Biosb Computational Metagenomics" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Biosb Computational Metagenomics
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Viromics
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/mdehollander/biosb-metagenomics" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    biosb-metagenomics
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Biosb Computational Metagenomics" class="md-nav__button md-logo" aria-label="Biosb Computational Metagenomics" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    Biosb Computational Metagenomics
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/mdehollander/biosb-metagenomics" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    biosb-metagenomics
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Day 1: Data processing
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Day 1: Data processing" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Day 1: Data processing
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/intro/" class="md-nav__link">
        Introduction
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/quality-control/" class="md-nav__link">
        Quality Control
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/assembly/" class="md-nav__link">
        Assembly
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../data-processing/binning/" class="md-nav__link">
        Binning
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
    
  
  
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3" type="checkbox" id="__nav_3" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_3">
          Day 3: Functional annotation
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Day 3: Functional annotation" data-md-level="1">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Day 3: Functional annotation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Viromics
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Viromics
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#step-1-initial-virus-discovery" class="md-nav__link">
    Step 1 - Initial virus discovery
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-2-run-checkv" class="md-nav__link">
    Step 2 - Run CheckV
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-3-taxonomic-classification" class="md-nav__link">
    Step 3 - Taxonomic classification
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-4-amg-detection" class="md-nav__link">
    Step 4 - AMG detection
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../bgcs/" class="md-nav__link">
        BGCs
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#step-1-initial-virus-discovery" class="md-nav__link">
    Step 1 - Initial virus discovery
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-2-run-checkv" class="md-nav__link">
    Step 2 - Run CheckV
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-3-taxonomic-classification" class="md-nav__link">
    Step 3 - Taxonomic classification
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#step-4-amg-detection" class="md-nav__link">
    Step 4 - AMG detection
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/mdehollander/biosb-metagenomics/edit/master/docs/functional-annotation/viromes.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="functional-annotation-viromics">Functional annotation - Viromics<a class="headerlink" href="#functional-annotation-viromics" title="Permanent link">&para;</a></h1>
<p>We will follow the virus discovery and AMG detection tutorial from the <a href="https://www.protocols.io/view/viral-sequence-identification-sop-with-virsorter2-5qpvoyqebg4o/">Sullivan lab</a> with the data from this course. In addition, we will do taxonomic classification of the discovered viruses.</p>
<h2 id="step-1-initial-virus-discovery">Step 1 - Initial virus discovery<a class="headerlink" href="#step-1-initial-virus-discovery" title="Permanent link">&para;</a></h2>
<p>First, we run VirSorter2 with a loose cutoff of 0.5 for maximal sensitivity. We are only interested in phages (dsDNA and ssDNA phage, this is also the default in Virsorter2). A minimal length 5000 bp is chosen since it is the minimum required by downstream viral classification. Note that the<code>--keep-original-seq</code> option preserves the original sequence of circular and (near) fully viral contigs (score &gt;0.8 as a whole sequence) and we are passing them to checkV to trim possible host genes left at ends and handle duplicate segments of circular contigs.</p>
<p><strong>Note:</strong> The following command takes about 2h. You can also continue with the results in <code>/data/precomputed/virome/vs2-pass1/</code>.</p>
<pre><code>virsorter run --keep-original-seq \
  -i /data/precomputed/assembly/final.contigs.fa \
  -w vs2-pass1 --min-length 5000 -j 2 all
</code></pre>
<p>You can find a description of the VirSorter2 output files <a href="https://github.com/jiarong/VirSorter2#detailed-description-on-output-files">here</a>. Look into <code>vs2-pass1/final-viral-score.tsv</code>.</p>
<details class="done">
<summary>How many phages of which viral group do you detect?</summary>
<p>1 ssDNA, 16 dsDNAphage</p>
</details>
<h2 id="step-2-run-checkv">Step 2 - Run CheckV<a class="headerlink" href="#step-2-run-checkv" title="Permanent link">&para;</a></h2>
<p>There could be some non-viral sequences or regions in the VirSorter2 results with a minimal score cutoff of 0.5. Here we use CheckV to quality control the VirSorter2 results and also to trim potential host regions left at the ends of proviruses. This command will take about 5 min to complete.</p>
<pre><code>checkv end_to_end vs2-pass1/final-viral-combined.fa checkv \
  -t 2 -d /data/databases/checkv-db-v1.4
</code></pre>
<p>The CheckV output is described <a href="https://bitbucket.org/berkeleylab/checkv/src/master/">here</a>. Look into <code>checkv/quality_summary.tsv</code>.</p>
<details class="done">
<summary>How many proviruses do you find and how many viruses?</summary>
<p>4 proviruses, 13 viruses</p>
</details>
<details class="done">
<summary>How many low, medium, and high quality viruses do you detect? How many of the viruses have direct terminal repeats?</summary>
<p>5 low,  3 medium, 7 high quality, of them 3 have DTR</p>
</details>
<p>We will work with both the detected viruses and proviruses, so we combine them:</p>
<pre><code>cat checkv/proviruses.fna checkv/viruses.fna &gt; checkv/combined.fna
</code></pre>
<h2 id="step-3-taxonomic-classification">Step 3 - Taxonomic classification<a class="headerlink" href="#step-3-taxonomic-classification" title="Permanent link">&para;</a></h2>
<p>We use vConTACT2 for taxonomic classification. This is done on the protein level and we first need to detect ORFs. This is done with prodigal:</p>
<pre><code>prodigal -p meta -a checkv/combined-prot.faa \
  -i checkv/combined.fna -f gff -o checkv/combined-prot.gff
</code></pre>
<p>Next we parse the prodigal output and prepare it for vConTACT2</p>
<pre><code>vcontact2_gene2genome -p checkv/combined-prot.faa \
  -o checkv/viral_genomes_g2g.csv -s 'Prodigal-FAA'
</code></pre>
<p>Finally, we are ready to cluster our sequences with the viral RefSeq.</p>
<p><strong>Note:</strong> The following command takes about 2h. You can also continue with the results in <code>/data/precomputed/virome/vcontact/</code>.</p>
<pre><code>vcontact2 --raw-proteins checkv/combined-prot.faa --rel-mode 'Diamond' \
  --proteins-fp checkv/viral_genomes_g2g.csv \
  --db 'ProkaryoticViralRefSeq94-Merged' --pcs-mode MCL \
  --vcs-mode ClusterONE --c1-bin /opt/conda/bin/clusterone \
  --threads 2 --output-dir vcontact
</code></pre>
<p>The vConTACT2 output is described <a href="https://bitbucket.org/MAVERICLab/vcontact2/wiki/Home">here</a>. You can load the file <code>genome_by_genome_overview.csv</code> in a spreadsheet program and check which one of the contigs are clustered (they are in the end of the file). Alternatively, you can check which of the clusters in <code>viral_cluster_overview.csv</code> contain contigs:</p>
<pre><code>grep k141 viral_cluster_overview.csv
</code></pre>
<details class="done">
<summary>Which of the contigs are clustered with phages from RefSeq? What can you say about the taxonomy and host range of the clusters?</summary>
<p>k141_124 - belongs to cluster VC_105 that has been split in 2 subclusters, the other subcluster contains Bacillus phages of the Myoviridae family<br />
k141_1485 - clusters with Bacillus phages of the Siphoviridae family<br />
k141_1484 - clusters with Escherichia phages of the Microviridae family<br />
k141_1004 - clusters with Stenotrophomonas phages of the Inoviridae family<br />
k141_292 - clusters with Sphingobium phages of the Siphoviridae family</p>
</details>
<h2 id="step-4-amg-detection">Step 4 - AMG detection<a class="headerlink" href="#step-4-amg-detection" title="Permanent link">&para;</a></h2>
<p>Then we run the checkV-trimmed sequences through VirSorter2 again to generate <code>affi-contigs.tab</code> files needed by DRAMv to identify AMGs. Note the<code>--seqname-suffix-off</code> option preserves the original input sequence name since we are sure there is no chance of getting &gt;1 proviruses from the same contig in this second pass, and the <code>--viral-gene-enrich-off</code> option turns off the requirement of having more viral genes than host genes to make sure that VirSorter2 is not doing any screening at this step.</p>
<p><strong>Note:</strong> The following command takes about 30 min. You can also continue with the results in <code>/data/precomputed/virome/vs2-pass2/</code>.</p>
<pre><code>virsorter run --seqname-suffix-off --viral-gene-enrich-off --provirus-off \
  --prep-for-dramv -i checkv/combined.fna -w vs2-pass2 --min-score 0.5 \
  -j 2 all
</code></pre>
<p>Then run DRAMv to annotate the identified sequences, which can be used for manual curation.</p>
<p><strong>Note:</strong> The following command takes about 10 min. You can run it or continue with the precomputed results in <code>/data/precomputed/virome/dramv-annotate/</code>.</p>
<pre><code>DRAM-v.py annotate -i vs2-pass2/for-dramv/final-viral-combined-for-dramv.fa \
  -v vs2-pass2/for-dramv/viral-affi-contigs-for-dramv.tab -o dramv-annotate \
  --skip_trnascan --threads 2 --min_contig_size 5000
</code></pre>
<p>Finally, we use <code>distill</code> to summarize the annotations.</p>
<pre><code>DRAM-v.py distill -i dramv-annotate/annotations.tsv -o dramv-distill
</code></pre>
<p>The summarized output can be found in <code>dramv-distill/amg_summary.tsv</code>.</p>
<details class="done">
<summary>Which contigs contain AMG and of which function? Which potential hosts have been identfied for them with the vConTACT2 analysis?</summary>
<p>We find 2 contigs that contain a dUTP pyrophosphatase: k141_124 (potential Bacillus phage) and k141_1483 (unclustered)</p>
</details>
<p>To get an overview of all AMGs detected in the data, you can check <code>dram-annotate/annotations.tsv</code>. The AMG flags (last column) are explained in the <a href="https://academic.oup.com/nar/article/48/16/8883/5884738">DRAM publication</a>.</p>
<p>You can also look into the original <a href="https://www.protocols.io/view/viral-sequence-identification-sop-with-virsorter2-5qpvoyqebg4o/">protocol</a> for considerations on manual curation.</p>

  <hr>
<div class="md-source-file">
  <small>
    
      Last update:
      <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">October 12, 2022</span>
      
    
  </small>
</div>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
        
        <a href="../../data-processing/binning/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Binning" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              Binning
            </div>
          </div>
        </a>
      
      
        
        <a href="../bgcs/" class="md-footer__link md-footer__link--next" aria-label="Next: BGCs" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              BGCs
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.expand"], "search": "../../assets/javascripts/workers/search.5bf1dace.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
  
      <script src="../../assets/javascripts/bundle.078830c0.min.js"></script>
      
    

  </body>
</html>